.vscode/launch.json

```diff
@@ -40,6 +49,7 @@
             "args": ["jwk", "refresh"],
             "env": {
                 "CONFIG_PATH": "${workspaceFolder}/configs",
+                "ENV_NAME": "local",
```

Comment:
This is removed with the refactor at https://github.com/podengo-project/idmsvc-backend/pull/178